_pages/fsa/install/building-the-fsa-storefront-from-libraries.md

```diff
@@ -68,11 +68,27 @@ yarn install
 ### Check app.module.ts for base URL and other settings ###
 
 Open the `src\app\app.module.ts` file, and check for any changes you want to make for your setup. 
+
+You should add following ngrx store module with specified runtime checks option in imports section:
```

Comment:
We should add the article, like this:
You should add **the** following ngrx store module with specified runtime checks option in **the** imports section:
Maybe imports should start with a capital I?